subprojects/core-api/src/main/java/org/gradle/api/file/ProjectLayout.java

```diff
@@ -38,6 +39,14 @@ public interface ProjectLayout {
      */
     Directory getProjectDirectory();
 
+    /**
+     * Returns the build root directory.
+     *
+     * @since 8.5
+     */
+    @Incubating
+    Directory getRootDirectory();
```

Comment:
I think `Build` should be included in this method name, since this is a `ProjectLayout`, but this method is the root dir of the build. With the current name, I would expect this to return the root directory of the project

Reply:
Or perhaps, we should instead expose a `BuildLayout` that has a `getRootDirectory`